xbmc/Application.cpp

```diff
@@ -563,6 +566,10 @@ bool CApplication::Create(const CAppParamParser &params)
     }
   }
 
+#if defined(TARGET_DARWIN_TVOS)
+  CTVOSInputSettings::GetInstance().Initialize();
```

Comment:
This seems perfect for the platform init

Reply:
Yes, this was my first try. But (maybe I'm wrong) user settings are only loaded **after** platform init call 😕.
See my recent messages in teamkodi channel.

Reply:
if the platform init is too early then what about the windowinging init?

Reply:
What do you mean?
Platform::Init() is called in ServiceManager::InitStageOne().
And ServiceManager::InitStageOne() is called in Application::Create() before SettingsComponent::Load().
This is why in this condition I can't put this line in Platform::Init(), because I need to access user settings.

Reply:
yes so what about putting it in the windowing init?

Reply:
Oh I understand your message now, will try. thank you 😉